Glossary: deprecate "Payload Body" per RFC 9110 (#37598)
mdn/content: files/en-us/glossary/payload_body/index.md

{{GlossarySidebar}}

> Prior to June 2022, the **payload body** or **payload** was the resource representation transferred in an HTTP message body, stripped of any [transfer encoding](/en-US/docs/Web/HTTP/Headers/Transfer-Encoding). With no transfer encoding, the _payload body_ and _message body_ are the same thing. Representation data could also be transferred in HTTP headers, but such data would not be regarded as a "payload".

Reviewer: ```suggestion
In HTTP messages, the **payload body** or **payload** was the resource representation transferred in a message body, stripped of any [transfer encoding](/en-US/docs/Web/HTTP/Headers/Transfer-Encoding). With no transfer encoding, the _payload body_ and _message body_ are identical. Representation data could also be transferred in HTTP headers, but such data would not be considered a "payload".
```

> With the release of RFC 9110, the word **content** replaces the word **payload**. See [**HTTP content**](/en-US/docs/Glossary/HTTP_Content).

Reviewer: ```suggestion
Since {{RFC("9110")}}, the word **content** replaces the word **payload**. See [**HTTP content**](/en-US/docs/Glossary/HTTP_Content).
```

{{GlossarySidebar}}

Prior to June 2022, the **payload body** or **payload** was the resource representation being transferred in an HTTP Message Body, stripped of any [transfer encoding](/en-US/docs/Web/HTTP/Headers/Transfer-Encoding). With no transfer encoding, the _payload body_ and _message body_ are the same thing. Representation data could also be transferred in the HTTP Header, but such data would not be regarded as "payload".

With the release of RFC 9110, the word **content** replaces the word **payload**. See [**HTTP content**](/en-US/docs/Glossary/HTTP_Content).

> For more information see [RFC 7230, section 3.3: Message Body](https://datatracker.ietf.org/doc/html/rfc7230#section-3.3), [RFC 7231, section 3: Representations](https://datatracker.ietf.org/doc/html/rfc7231#section-3), and [RFC 9110, appendix B.3: Changes from RFC 7231](https://datatracker.ietf.org/doc/html/rfc9110#name-changes-from-rfc-7231).

Reviewer: Let's use a See also section like the other entry:

```suggestion
## See also

- {{Glossary("Content header")}}
- [RFC 9110, section 6.4: Content](https://httpwg.org/specs/rfc9110.html#rfc.section.6.4) (obsoletes [RFC 7231](https://datatracker.ietf.org/doc/html/rfc7231#section-3.3) Payload Semantics)
  - [Changes from RFC 7231](https://httpwg.org/specs/rfc9110.html#changes.from.rfc.7231)
```
